### PR TITLE
Fixes #20: Test Chat Consumers

### LINKED ISSUE #20: Testing chat consumers

### PR DESCRIPTION
**Pull request by Alir3z4**
Fixes #20

### CODE REVIEW

File: fabfile/fab_assert.py [hunk 1]
```diff
@@ -104,10 +104,8 @@ def coverage() -> None:
     if os.path.isfile(COVERAGE_REPORT_FILE):
         os.remove(COVERAGE_REPORT_FILE)
 
-    initialize_db()
-    local("coverage run --source='.' manage.py {0}".format(get_test_command(
-        parallel=False, fake_migrations=False
-    )))
+    # initialize_db()
```
Comment (AmirSavand) [inline fabfile/fab_assert.py]: Why?

Comment (Alir3z4) [reply fabfile/fab_assert.py]: The whole thing will be using in memory SQLite database.
The initialize_db() method is used to initialize a PostrgeSQL database.

**Comment by AmirSavand** [comment]
This is gold!

**Comment by AmirSavand** [comment]
Yes, we might need to change our error handing, but in a far future.

**Comment by Alir3z4** [comment]
Yess!
In the websocket itself, I'll add users in a room etc

**Pull request merged**